### DISCUSSION TITLE
[FIRMWARE REQUEST] Lekker switch compatibility

### DISCUSSION THREAD

**Discussion by ZoddoDK** [OpenStickCommunity/GP2040-CE#385] [Ideas]
Would it be possible to make Lekker switches work on a Pico Fighting board? It would be amazing to have a hitbox with the great functions like rapid Trigger, and manual actuation points to work on a Hitbox/arcade stick.

**Comment by ioridev** [comment]
It seems that the Raspberry Pi Pico only has 3 ADC pins that allow analog input, so getting the Lekker switch to work is not going to be easy. Also, the ADC on the RP2040 seems to have errata and accuracy issues. 

p629
https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf

**Comment by TheTrainGoes** [comment]
Hi @ZoddoDK ,

This request is out of scope for the current project due to available ADC pins.
